content/doc/developer/tutorial/create.adoc

```diff
@@ -36,37 +36,43 @@ Choose a number or apply filter (format: [groupId:]artifactId, case sensitive co
 Choose io.jenkins.archetypes:hello-world-plugin version:
 1: 1.1
 2: 1.2
-Choose a number: 2: *2* // <2>
+3: 1.3
+Choose a number: 3: *3* // <2>
 [INFO] Using property: groupId = unused
 Define value for property 'artifactId': *demo* // <3>
 Define value for property 'version' 1.0-SNAPSHOT: : // <4>
-[INFO] Using property: package = org.jenkinsci.plugins.sample
+[INFO] Using property: package = io.jenkins.plugins.sample
 Confirm properties configuration:
 groupId: unused
 artifactId: demo
 version: 1.0-SNAPSHOT
-package: org.jenkinsci.plugins.sample
+package: io.jenkins.plugins.sample
  Y: : *y* // <5>
 
 ----
 <1> Enter the number for the +hello-world-plugin+ archetype, *3* in this case.
-<2> This tutorial is based on version 1.2 of the *hello-world-plugin* archetype, so enter *2* to select it.
+<2> This tutorial is based on version 1.3 of the *hello-world-plugin* archetype, so enter *3* to select it.
 <3> This is mandatory and uniquely identifies your plugin in Jenkins.
     This plugin tutorial uses the name `demo` (user input highlighted in bold).
     If you want to publish your plugin, make sure the name is not already taken, and that the name you choose is future-proof:
-    The `artifactId` cannot be changed after you've published your first release.
+    the `artifactId` cannot be changed after you've published your first release.
+    Do _not_ use the words `jenkins` or `plugin` in this ID—only the words describing what kind of Jenkins plugin this is.
```

Comment:
Best to warn people early…

Reply:
Not a huge deal since without release it's just a POM entry and possibly a package rename, but doesn't hurt either.